site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,127 @@
+# Cross namespace references and ReferencePolicy
+
+### Terminology note
+When discussing the process of creating cross-namespace object references, this
+document and the documentation on the API itself refers to the object being
+referred to as "the referent object", using the meaning of "referent" to be
+"the person, thing, or idea that a word, phrase, or object refers to".[1](https://dictionary.cambridge.org/dictionary/english/referent)
```

Comment:
This footnote looks a bit off in the deploy preview. Maybe either use `<sup>` tag or a full word like `(source)`.
https://deploy-preview-741--kubernetes-sigs-gateway-api.netlify.app/v1alpha2/references/cross-namespace-references/